WrightTools/units.py

```diff
@@ -111,86 +113,71 @@ def converter(val, current_unit, destination_unit):
         pass
     else:
         warnings.warn('conversion {0} to {1} not valid: returning input'.format(
-                       current_unit, destination_unit))
+                      current_unit, destination_unit))
     return val
 
 
-def kind(units):
-    """Find the kind of given units.
+def get_symbol(units):
+    """Get default symbol type.
 
     Parameters
     ----------
-    units : string
-        The units of interest
+    units_str : string
+        Units.
 
     Returns
     -------
     string
-        The kind of the given units. If no match is found, returns None.
+        LaTeX formatted symbol.
     """
-    for d in unit_dicts:
-        if units in d.keys():
-            return str(d['kind'])
-
-
-# --- symbol --------------------------------------------------------------------------------------
-
-
-class SymbolDict(dict):
-    """Subclass dictionary to get at __missing__ method."""
-
-    def __missing__(self, key):
-        """Define what happens when key is missing."""
-        return self['default']
-
-
-# color
-color_symbols = SymbolDict()
-color_symbols['default'] = r'E'
-color_symbols['nm'] = r'\lambda'
-color_symbols['wn'] = r'\bar\nu'
-color_symbols['eV'] = r'\hslash\omega'
-color_symbols['Hz'] = r'f'
-color_symbols['THz'] = r'f'
-color_symbols['GHz'] = r'f'
+    if kind(units) == 'energy':
+        d = {}
+        d['nm'] = r'\lambda'
+        d['wn'] = r'\bar\nu'
+        d['eV'] = r'\hslash\omega'
+        d['Hz'] = r'f'
+        d['THz'] = r'f'
+        d['GHz'] = r'f'
+        return d.get(units, 'E')
+    elif kind(units) == 'delay':
+        return r'\tau'
+    elif kind(units) == 'fluence':
+        return r'\mathcal{F}'
+    elif kind(units) == 'pulse_width':
+        return r'\sigma'
+    else:
+        return ''
```

Comment:
Should the default be the unit itself?

Reply:
:thinking: 

probably not? because the unit appears elsewhere in the label

actually, the kind itself might be a good default